modules/ingest-common/src/test/java/org/elasticsearch/ingest/common/CsvProcessorTests.java

```diff
@@ -158,7 +218,7 @@ public void testEmptyFields() throws Exception {
         items.keySet().stream().skip(numItems - 1).forEach(key -> assertFalse(ingestDocument.hasField(key)));
     }
 
-    public void testWrongStings() throws Exception {
+    public void testWrongStings() {
```

Comment:
typo: `Sting` -> `String`

(But it was there before and you've got a green build so if no other changes are required then no need to trigger a rebuild just for this.)